bin/find-graphql-files.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+GRAPHQL_FILE_LIST="$PROJECT_DIR/KsApi/GraphQLFiles.xcfilelist"
+
+#Find all GraphQL files in KsApi.
+#Replace the literal $PROJECT_DIR with the token $(PROJECT_DIR)
+find $PROJECT_DIR -name "*.graphql" | sed "s#^$PROJECT_DIR#\$(PROJECT_DIR)#" > "$GRAPHQL_FILE_LIST.tmp"
```

Comment:
Replacing the actual project directory (like `/Users/amy/ios-oss/`) with the token means we're not always overwriting this file for one another.